cmds/core/mv/mv_test.go

```diff
@@ -281,6 +282,42 @@ func TestMvNoClobber(t *testing.T) {
 	}
 }
 
+func TestCrossFileSystems(t *testing.T) {
+	// The namespace creator does not have a teardown capability and it looked a bit
+	// tricky to add one.
+	if os.Getuid() != 0 {
```

Comment:
testutil.SkipIfNotRoot(t)

pkg/testutil

but I'd say it's optional, just a nit